Assets/Scripts/Player/Health.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//Generally used to calculate the damage distributed to certain body parts and the status of said body parts once a threshold is broken
+
+public class Health : MonoBehaviour
+{
+    //current entity type (0 being organic based on enum)
+    public Entities currentEntity;
+
+    public int maxHealth = 100;
+    public int currentHealth;
+
+    //sets the max blood capacity to 560ccs
+    private float maxBloodVol = 560f;
+    private float currentBloodVol;
+
+    //number of limbs that are currently bleeding
+    private int numBleedingLimbs;

```

Comment:
This should be a list that stores references to the limbs that are bleeding. You can get an integer value from this by using the .len of the list.